src/app.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
why is this added?

Reply:
🤦🏽‍♂️  it's because of difference in our linebreak char code. My linter always marks the line-break as an error. That is just to make it stop

Reply:
go ahead and delete this line and pay no attention to the lint error for now. I will convert everything to LF and the error will go away for you.

Reply:
👌🏽